server/db/models/order.js

```diff
@@ -0,0 +1,20 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Order = db.define('order', {
+  status: {
+    type: Sequelize.ENUM('Created', 'Processing', 'Cancelled', 'Completed' )
+  },
+  guestId: {
```

Comment:
Should there be a userId field here? We could have both a userId and guestId, and we could allow null for both.